docs/canvas/canvas-common-functions.asciidoc

```diff
@@ -6,640 +6,765 @@ The common functions can run anywhere, which means they'll execute wherever
 
-*Accepts:* datatable
+*Accepts:* `null`
 
 [cols="3*^<"]
 |===
 |Argument |Type |Description
 
-|show
-|boolean
-|default: `true`
+|`show`
+|`boolean`
+|Show the axis labels? 
+
+Default: `true`
+
+|`position`
+|`string`
+|The position of the axis labels, for example, top, bottom, left, and right 
+
+Default: `"left"`
+
+|`min`
+|`number`, `string`, `null`
+|Minimum value displayed in the axis. Must be a `number`, a date in milliseconds since epoch, or an ISO8601 `string`
+
+|`max`
+|`number`, `string`, `null`
+|Maximum value displayed in the axis. Must be a `number`, a date in milliseconds since epoch, or an ISO8601 `string`
 
-|position
-|string
-|The position of the axis labels, for example, top, bottom, left, and right
+|`tickSize`
+|`number`, `null`
+|Increment size between each tick. Use for `number` axes only.
 |===
 
-*Returns:* axisConfig
+*Returns:* `axisConfig`
 
 
 [float]
-=== case
+[[case_fn]]
+=== `case`
 
-Builds a case (including a condition/result) to pass to the switch function.
+Builds a `case` (including a condition/result) to pass to the <<switch_fn>> function.
 
-*Accepts:* Anything or nothing
+*Accepts:* `any`
 
 [cols="3*^<"]
 |===
 |Argument |Type |Description
 
-|_Unnamed_
-|any
-|This value is compared to the context to see if the condition 
-s met. It is overridden by the "if" argument if both are provided.
+|_Unnamed_ 
+
+Alias: `when`
+|`any`
+|This value is compared to the _context_ to see if the condition 
+is met. It is overridden by the "if" argument if both are provided.
 
-|if*
-|boolean
+|`if`
+|`boolean`
 |This value determines whether or not the condition is met. 
 It overrides the unnamed argument if both are provided.
 
-|then
-|any
+|`then` ***
+|`any`
 |The value to return if the condition is met
 |===
 
-*Returns:* case
+*Returns:* `case`
+
 
 [float]
-=== clog
+[[clear_fn]]
+=== `clear`
 
-Outputs the context to the console.
+Clears _context_ and returns `null`
 
-*Accepts:* Anything or nothing
+*Accepts:* `null`
 
-*Returns:* Depends on your input and arguments
+*Returns:* `null`
 
 
 [float]
-=== columns
+[[columns_fn]]
+=== `columns`
 
 Includes or excludes columns from a data table. If you specify both, this will exclude first.
 
-*Accepts:* datatable
+*Accepts:* `datatable`
 
 [cols="3*^<"]
 |===
 |Argument |Type |Description
 
-|include
-|string
+|`include`
+|`string`
 |A comma separated list of column names to keep in the table 
-(default: `null`)
 
-|exclude
-|string
+|`exclude`
+|`string`
 |A comma separated list of column names to remove from the table 
-(default: `null`)
 |===
 
-*Returns:* datatable
+*Returns:* `datatable`
 
    
 [float]
-=== compare
+[[compare_fn]]
+=== `compare`
 
-Compares the input to something else to determine true or false. 
-Usually used in combination with `if`. This only works with primitive types, 
-such as number, string, and boolean.
+Compares the input to something else to determine `true` or `false`. 
```

Comment:
How about:

`To determine `true` or `false`, compares the input. TIP: The `compare` function works only with primitive types, such as `boolean`, `number`, and `string`. Use `compare` with `<<if_fin>>. See also <<eq_fn>>, <<gt_fn>>, <<gte_fn>>, <<lt_fn>>, <<lte_fn>>, and <<neq_fn>>.`